pkgs/applications/video/anime-downloader/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, python3Packages, aria2, mpv, nodejs, fetchFromGitHub }:
+with python3Packages;
```

Comment:
```suggestion

```

Reply:
Be explicit on the Python extensions you are using.

It is bad practice to bring whole things explicitly into the expression.